package.json

```diff
@@ -23,12 +23,13 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-jobs",
   "dependencies": {
-    "async": "^1.4.2",
-    "moment-interval": "git://github.com/digitalbazaar/moment-interval"
+    "async": "^2.6.0",
+    "moment-interval": "git://github.com/digitalbazaar/moment-interval",
+    "uuid": "^3.2.1"
   },
   "peerDependencies": {
-    "bedrock": "^1.0.0",
-    "bedrock-mongodb": "^3.0.0"
+    "bedrock": "^1.9.2",
+    "bedrock-mongodb": "^5.1.2"
```

Comment:
I think this can be a bit more loose and use "3.x - 5.x".

Reply:
@davidlehn can you make a case for doing this?  After giving it some thought, I can think of no practical advantage.  Here are some of the reasons against:
- We have no way to test with older versions of deps.
- Without a way to test with older dep, how would a dev know if they have added code that results in a failure in an older version.
- If the package file indicates that something should work with versions 3-5, then it *should* work with versions 3-5 and we have no way to guarantee that.
- I'm sure it is not our intention to backport features/fixes to those older versions
- I don't know who the beneficiary of this more loose approach would be.
- Maintaining package files is hard enough as it is, I fee that this adds even more complexity and unreliability into the mix.

Reply:
We are discussing a situation that will/will not cause `npm` to generate a dependency warning right?  So if someone chooses to continue using bedrock-foo@old.x they would just get an npm warning?  In which case they would be made aware that they are not running the current version.

Reply:
Idea was that if we think older bedrock-mongodb still works then to allow it.  True that we are not testing all version combinations, but we never really are.  We're just trusting to update deps as needed when we know we make breaking changes or use new apis.  The 4.x and/or 5.x upgrades are breaking without doing some sort of database migration so it seemed like a reasonable idea to indicate the old versions still would work.  The top level package still has to pick which version to install to satisfy the peer dep.  This is mostly to allow older code that has a dependency path to an older version to keep working without all the npm warnings/errors.  There were so many it was too difficult to see real problems in all the noise.  I updated a ton of other packages using this same style since it was the same issue everywhere.